tasks/run.js

```diff
@@ -39,3 +24,25 @@ async function runTask (name, args = []) {
     task.on('close', resolve)
   })
 }
+
+/**
+ * Creates a Gulp task for runTask()
+ *
+ * @param {string} name - npm script name
+ * @param {string[]} [args] - npm script arguments
+ * @returns {() => Promise<number>} Exit code
+ */
+const runTask = (name, args = []) => {
+  const script = `${name} ${args.join(' ')}`.trim()
+
+  // Create named runner
+  // https://gulpjs.com/docs/en/api/task#task-metadata
+  const runner = () => task(name, args)
+  runner.displayName = `npm run ${script}`
+
+  return runner
+}
```

Comment:
I'm a bit confused by the naming. From what I understand, this'll only run npm scripts, so maybe we should clarify this by renaming to `runNpmScript` ?

Reply:
Mind if I use the following?

**Function for npm script**
`task()` → **`npmScript()`**

**Function for gulp task** (that returns npm script)
`runTask()` → **`npmScriptTask()`**